app/youtube-grabber.js

```diff
@@ -114,7 +114,13 @@ class YoutubeGrabber {
     if (channelHeaderData.badges) {
       isVerified = channelHeaderData.badges.some((badge) => badge.metadataBadgeRenderer.tooltip === 'Verified')
     }
-
+    let tags=null;
+    try {
```

Comment:
Please try to find out up to which properties are always available, so we can avoid using try catch. Because then an if statement can be used in combination with the property check `'x' in channelPageResponse[y]......`

Reply:
I'm not sure what I was thinking when I originally coded this (I didn't want an error to occur if YT updated their api but then it would be harder to tell if an error actually occurred). It should be good to go now